### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by djo1975**
- We got a database diagram, interpret it:
clinic_diagram
- We made schema_based_on_diagram.sql based on the diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  id SERIAL PRIMARY KEY, 
+  name VARCHAR(250) NOT NULL, 
+  date_of_birth DATE) NOT NULL;
+
+CREATE TABLE invoices(
+  id SERIAL PRIMARY KEY, 
+  total_amount DECIMAL(5, 2) NOT NULL, 
+  generated_at TIMESTAMP NOT NULL, 
+  payed_at TIMESTAMP NOT NULL, 
+  medical_history_id INT NOT NULL
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql:24]: - As is on the diagram, kindly add the FOREIGN KEY with REFERENCES missing on the table, with medical_histories(id) from the invoices table

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  )
+
+create table treatments (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR(250) NOT NULL,
+  name VARCHAR(250) NOT NULL,
+)
+
+create table invoice_items (
+  id primary key,
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql:45]: - As is on the diagram, kindly add the FOREIGN KEY with REFERENCES missing on the table, with treatments(id) from the treatments table
- As is on the diagram, kindly add the FOREIGN KEY with REFERENCES missing on the table, with invoices(id) from the invoices table

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  unit_price DECIMAL(5, 2) NOT NULL,
+  quantity INT NOT NULL,
+  total_price DECIMAL(5, 2) NOT NULL,
+  invoice_id INT NOT NULL,
+  treatment_id INT NOT NULL
+)
+
+
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql]: As it´s not shown on the diagram, there´s one missing table that is created from the relationship between treatments and medical_histories tables.

- Kindly create this table, maybe you can name it treatments_histories 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @djo1975 

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

***While you did a great job creating the tables, there's some information missing from the diagram provided***

cat-driving-serious

## Highlights

✔️ Good commit history
✔️ Good work creating tables
✔️ README looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @djo1975 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes
\

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by djo1975** [comment]
THX :)

**Pull request merged**